Homework5/Person.cs

```diff
@@ -12,5 +12,18 @@ public class Person
         public string Name { get; set; }
         public int Age { get; set; }
         public List<Pet> Pets { get; set; }
+
+
+        public void UpdatePerson(Person newDetails) {
+            if (Id != newDetails.Id) Id = newDetails.Id;
+
+            if (SpouseId != newDetails.SpouseId) SpouseId = newDetails.SpouseId;
+
+            if (Name != newDetails.Name) Name = newDetails.Name;
+
+            if (Age != newDetails.Age) Age = newDetails.Age;
+
+            if (Pets != newDetails.Pets) Pets = newDetails.Pets;
```

Comment:
You can make it more simple- just assign the value. If it is not the same- it will be the same. If it was the same- nothing will change when you set the same value to the same value.

Reply:
So remove the ifs.